src/note/note.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable, Logger } from '@nestjs/common';
+import {InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { notes } from './note.entity';
+import { Note } from '../interfaces/note'
+@Injectable()
+export class  NoteService {
+constructor(
+@InjectRepository(notes)
+private noteRepository: Repository<notes>,
+) {}
+async index(): Promise<Note[]> {
+return await this.noteRepository.find();
+}
+async findOne(id): Promise<Note[]>{
+return await this.noteRepository.find({where: [{ 'id': id }]});
+}
+async create(note: Note): Promise<any> {
+ const noteReady = await this.noteRepository.save(note);
+// Logger.log(noteReady);
+}
+async updata(id: string, body: Note): Promise<any>{
+const objAnswer =  await this.noteRepository.update(id,body);
```

Comment:
tak jak w przypadku post'a, 

const objAnswer =  await this.noteRepository.update(id,body); do zmiany na 

return this.noteRepository.update(id,body);